defactor-com/documentation
# Feat/update counterparty pools contract documentation 97

### Update Counter Party Pool documentation

### What does this PR do?

- Resolve #97 
- Update function changes, add new ones, and flowcharts 

### Steps to test

1. Run the project locally
2. Go to Pools > Smart Contracts > Counter Party Pool Contract
3. Check every subsection changes

#### CheckList
